app/shared/src/__tests__/Price.test.js

```diff
@@ -2,25 +2,30 @@
 
 import * as React from 'react';
 import ShallowRenderer from 'react-test-renderer/shallow';
+import testRenderer from 'react-test-renderer';
 
 import Price from '../Price';
+import CurrencyFormatter from '../currency/CurrencyFormatter';
 
-const renderer = new ShallowRenderer();
+jest.mock('../currency/CurrencyFormatter', () => jest.fn());
```

Comment:
Why not to mock `RNCurrencyManager` like every other native module?

Reply:
The RNCurrencyManager mocks ´NativeModules.RNCurrencyManager`. Here I want to test that we actually call the module in `app/shared/src/currency/CurrencyFormatter`